Source/WebCore/accessibility/atspi/AccessibilityObjectAtspi.h

```diff
@@ -36,6 +36,8 @@ namespace WebCore {
 class AXCoreObject;
 class AccessibilityRootAtspi;
 
+using RelationMap = HashMap<Atspi::Relation, Vector<RefPtr<AccessibilityObjectAtspi>>, IntHash<Atspi::Relation>, WTF::StrongEnumHashTraits<Atspi::Relation>>;
```

Comment:
Good, but either (a) move it inside the class so it's scoped to AccessibilityObjectAtspi, or (b) name it something like AtspiRelationMap.